docs/overview.rst

```diff
@@ -15,52 +15,68 @@ abilities. It is meant to be **easy to use** and **easy to self-host**.
 Why use Kinto?
 ==============
 
-We believe data belongs to the users, and not to the application authors. When
-writing applications, data should be made available to any device, connected or
-offline, and everything should be in sync.
-
-Rather than spending a non-trivial amount of time and expertise on implementing
-all that (and then maintaining it!), you could use Kinto, that does all that
-for you:
-
-- Expose your data over an HTTP interface, unlike databases like PostgreSQL
-- Use simple HTTP requests rather than SQL
-- Use `Kinto.js <https://kintojs.readthedocs.io/en/latest/>`_ to easily
-  implement offline first clients
-- Choose the database you want from those that Kinto supports, and use a
-  unified API to access its data
-- Manage your data using the handy
-  `admin UI <http://kinto.github.io/kinto-admin/>`_
-- Easily
-  :ref:`set up live push notifications <tutorial-notifications-websockets>`
-  for live updates of your application
-- Make it possible to
-  :ref:`share data between users <api-permissions>`
-  (using fine-grained permissions)
-- Take advantage of
-  :ref:`schema validation <collection-json-schema>`
-  if you need it
-- :ref:`Track changes <api-history>` on your application data
+    Don't build silos. Redecentralize the Web!
+
+It's hard for frontend developers to respect users privacy when building applications
+that work offline, store data remotely, and synchronise accross devices.
+Existing solutions either rely on big corporations that crave for user data, or require
+a non-trivial amount of time and expertise to setup a new server for every new project.
+
+We want to help developers focus on the frontend, and we don't want the challenge
+of storing user data to get in their way! The path between a new idea and starting into
+production should be short!
+
+Also, we believe data belongs to the users, and not to the application authors.
+Applications should be decoupled from the storage location, and users should be
+able to choose where their personal data are stored.
+
+We envision mutualisation and self-hosting: the backend is deployed, secured and scaled
+only once and efforts are capitalized.
 
 
 .. _use-cases:
 
 Use cases
 =========
 
-- A generic Web database for frontend applications.
-- Build collaborative applications with fine-grained permissions.
-- Store encrypted data at a location you control.
-- Synchronise application data between different devices.
+- **A generic Web database**: for mobile and Web apps, games, or IoT...
+- **Quickly prototype frontend applications**: don't loose time with server stuff.
```

Comment:
Makes me think of a tagline so I'm putting this here: *the backend for your next single page app is already there*